bench/workgen/runner/compress_ratio.py

```diff
@@ -121,7 +128,7 @@ def operations(optype, tables, key, value = None, ops_per_txn = 0, logtable = No
 workload = Workload(context, threads)
 workload.options.run_time = 60
 workload.options.report_interval = 1
-workload.options.sample_interval = 1
+workload.options.sample_interval_ms = 1000
```

Comment:
I don't see any usages of this in the diff. Is that ok?

Reply:
`workgen.cxx` and `wtperf.py` have code to check the `sample_interval_ms` value. And in below code segement ms and sec intervals are used together to control the sleep time:
>     // The whole and fractional part of sample_interval are separated,
>     // we don't want to sleep longer than a second.
>     int sample_secs = ms_to_sec(options->sample_interval_ms);
>     useconds_t sample_usecs =
>       ms_to_us(options->sample_interval_ms) - sec_to_us(sample_secs);
Not sure if the change here makes any logic difference though.

Reply:
Thanks @tetsuo-cpp and @lukech, you are right, the change to use sample_interval_ms is a no-op.  I mistakenly thought I had deprecated sample_interval earlier, I hadn't.  So I'll change back to use sample_interval for clarity.